doc/guides/backport-to-lts-branches.md

```diff
@@ -0,0 +1,124 @@
+# How to backport to LTS branches
```

Comment:
Should this be called "Creating a release"  instead.

Reply:
+1 to preparing an LTS release or `Triaging for LTS releases`